### PR TITLE
Add cleveldb.Stats()

### PR DESCRIPTION
**Pull request by jackzampolin**
Fixes: https://github.com/tendermint/tendermint/issues/3378

### CODE REVIEW

File: libs/db/c_level_db.go [hunk 1]
```diff
@@ -129,7 +129,16 @@ func (db *CLevelDB) Print() {
 // Implements DB.
 func (db *CLevelDB) Stats() map[string]string {
 	// TODO: Find the available properties for the C LevelDB implementation
```
Comment (melekes) [inline libs/db/c_level_db.go]:
```suggestion
```

**Comment by melekes** [comment]
Where did you find the properties?

**Comment by melekes** [comment]
@jackzampolin Thanks for contributing 👍

**Comment by jackzampolin** [comment]
Thank you for finishing this up for me @melekes!! 🙇

**Pull request merged**